venv/10_matches.py

```diff
@@ -0,0 +1,38 @@
+Напрасно подумали бы, что это есть насмешка, карикатура исторических описаний. Напротив, это есть самое мягкое выражение тех противоречивых и не отвечающих на вопросы ответов, которые дает вся история, от составителей мемуаров и историй отдельных государств до общих историй и нового рода историй культуры того времени.
+Странность и комизм этих ответов вытекают из того, что новая история подобна глухому человеку, отвечающему на вопросы, которых никто ему не делает.
+Если цель истории есть описание движения человечества и народов, то первый вопрос, без ответа на который все остальное непонятно, — следующий: какая сила движет народами? На этот вопрос новая история озабоченно рассказывает или то, что Наполеон был очень гениален, или то, что Людовик XIV был очень горд, или еще то, что такие-то писатели написали такие-то книжки.
+Все это очень может быть, и человечество готово на это согласиться; но оно не об этом спрашивает. Все это могло бы быть интересно, если бы мы признавали божественную власть, основанную на самой себе и всегда одинаковую, управляющею своими народами через Наполеонов, Людовиков и писателей; но власти этой мы не признаем, и потому, прежде чем говорить о Наполеонах, Людовиках и писателях, надо показать существующую связь между этими лицами и движением народов.
+Если вместо божественной власти стала другая сила, то надо объяснить, в чем состоит эта новая сила, ибо именно в этой-то силе и заключается весь интерес истории.
+История как будто предполагает, что сила эта сама собой разумеется и всем известна. Но, несмотря на все желание признать эту новую силу известною, тот, кто прочтет очень много исторических сочинений, невольно усомнится в том, чтобы новая сила эта, различно понимаемая самими историками, была всем совершенно известна.""".lower()
+# signs_list = ['.', ',', '?', '!', '-', ';', ':', '...', '"', "'", '"""', ')', '(', '–', '—']
+for temp in string.punctuation:
+    if temp in my_str:
+        my_str = my_str.replace(temp, '').strip()
+        res_str = my_str.split()
```

Comment:
Не самый быстрый вариант. Вот как можно было сделать по другому. И список, кстати, совсем не нужен.
```suggestion
res_str = ''.join(filter(lambda x: x not in string.punctuation, my_str.lower()))
```